firmware/application/radio.cpp

```diff
@@ -157,13 +157,14 @@ bool set_tuning_frequency(const rf::Frequency frequency) {
 
 void set_rf_amp(const bool rf_amp) {
 	rf_path.set_rf_amp(rf_amp);
-	
+	/*
 	if (direction == rf::Direction::Transmit) {
```

Comment:
Why is this commented out?

Reply:
These code seems meaningless...
There's no reason to turn tx_led on when turning amp on.

Reply:
That might be the case but this function might be called in other instances. I need to check if it actually  affects something else but for this PR seems unrelated to comment this, don't you agree?

Reply:
Yes... Maybe I will send another PR.